package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@segment/integration-intercom",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
hmmm but feature wise nothing is changing? The only "migration" we are doing is adding a new metadata setting. 

> the changes are not simply isolated in the integration code.

What do you mean?

also cc @tsholmes on this! I'm happy to change this to whatever needs to be, just playing some devil's advocate 😈